packages/auth/src/authClient.ts

```diff
@@ -419,6 +465,8 @@ export class AuthClient {
       stamper,
       orgId,
       idToken,
+      bundle,
+      authType: authType as AuthType,
```

Comment:
...then this will work:
```suggestion
      authType,
```

Reply:
genius (brb looking into `Omit` vs `Exclude`)

Reply:
`Omit` is to remove a specific property from an object.
`Exclude` is to remove a member from a union.